apps/taquito-test-dapp/src/tests.ts

```diff
@@ -699,6 +727,8 @@ export const init = (
     id: "permit",
     name: "Permit contract",
     description: "This test implements TZIP-17",
+    documentation: '',
```

Comment:
The link makes the page reload. Can it be removed as there is no documentation related?